consensus/hotstuff/consumer.go

```diff
@@ -133,14 +133,6 @@ type Consumer interface {
 	// and must handle repetition of the same events (with some processing overhead).
 	OnQcIncorporated(*flow.QuorumCertificate)
```

Comment:
Previously this one was emitted by `ForkChoice`, we don't have that anymore, we could transfer this responsibility to `Forks` to notify about every QC incorporated in `AddBlock` function. We could also leave it for later and have a separate issue to update/extend/remove all metrics.

Reply:
From what I can tell, it's not used at all currently. My gut feeling would be to remove it, if we don't have a specific use case for making use of the notifications.

Reply:
+1